virt/virtual_machines/advanced_vm_management/virt-about-multi-queue.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-about-multi-queue"]
+= About multi-queue functionality
+include::_attributes/common-attributes.adoc[]
+:context: virt-about-multi-queue
+
+toc::[]
+
+Use multi-queue functionality to scale network throughput and performance on virtual machines (VMs) with multiple vCPUs.
+
+By default, queueCount, which is derived from the domain XML, is determined by the number of vCPUs allocated to a VM. Network performance does not scale as the number of vCPUs increases. Additionally, because virtio-net has only one TX and RX queue, guests cannot transmit or retrieve packs in parallel.
```

Comment:
* Is "queueCount" a proper noun or is it something that should go in backticks and have a description (idk what that would be). Maybe:
```suggestion
By default, the `queueCount` value, which is derived from the domain XML, is determined by the number of vCPUs allocated to a VM. Network performance does not scale as the number of vCPUs increases. Additionally, because virtio-net has only one TX and RX queue, guests cannot transmit or retrieve packs in parallel.
```

* Do TX and RX need definitions here?
* Wonder about "virtio-net" as well - does it need backticks? (I don't know enough to say :slightly_smiling_face:)

Reply:
Will update formatting of queueCount. AFAIK virtio-net is the name of the feature but not the value, so leaving as-is. 

@yossisegev Do you think we need to define RX/TX or should users be familiar with the retrieve/transmit definitions?

Reply:
Rx/Tx are well-known terms, I don't think they need any definition.
I would, however, change it from `RX` and `TX` to `Rx` and `Tx` (lower-case 'x'), which is the standard use.

Reply:
Will update format.